packages/core/src/components/dropdown/dropdown.scss

```diff
@@ -23,13 +23,32 @@
   .dropdown-select {
     position: relative;
 
+    button:focus {
+      border-bottom-color: var(--tds-dropdown-border-bottom);
```

Comment:
I haven't been part of the discussion around these updated styles, should the dropdown not have any blue border when focused like the input field for number of pages?

Reply:
Good call. Designs for this are in "WIP" so we do guesses here, but I will take it on myself and add a blue outline like the input on the right side has.